test/tests/ver-1ja/ver-1ja.i

```diff
@@ -0,0 +1,89 @@
+# Verification Problem #1a from TMAP7 V&V document
+# Radioactive Decay of Mobile Tritium in a Slab
+
+# Case and model parameters (adapted from TMAP7)
+tritium_concentration_initial = 1.5e5 # atoms/m3
```

Comment:
```suggestion
tritium_concentration_initial = ${units 1.5e5 atoms/m3}
```
This is a style choice potentially, but doing the documentation this way makes it double as an easy way to change units down the line if we want to modify the same input file.